index.html

```diff
@@ -2,45 +2,265 @@
-            <li><a href="#"><img src="Images/mediummedium.png" alt=" Medium logo"></a></li>
-            <li><a href="#"><img src="Images/githubgithub.png" alt="Github logo"></a></li>
-           <li><a href="#"><img src="Images/angellistangelist.png" alt="Angellist logo"></a></li>
-            </ul>
+      </div>
+      <div class="images">
+        <ul>
+          <li> <a href="#"><img class="img" src="Images/twittertwitter.png" alt="Twitter logo"></a></li>
+          <li><a href="#"><img class="img" src="Images/Linkedin.png" alt="LinkedIn logo"></a></li>
+          <li><a href="#"><img class="img" src="Images/mediummedium.png" alt=" Medium logo"></a></li>
+          <li><a href="#"><img class="img" src="Images/githubgithub.png" alt="Github logo"></a></li>
+          <li><a href="#"><img class="img" src="Images/angellistangelist.png" alt="Angellist logo"></a></li>
+        </ul>
+      </div>
+      <div class="space"></div>
+      <div>
+        <img class="drop-down" src="Images/drop-down.png" alt="drop-down button">
+      </div>
+    </section>
+  </main>
+  <section class="rest">
+    <h1 class="works">My Recent Works</h1>
+    <div class="indicator"> <img id="line" src="Images/Indicator.png" alt="fancyLine"></div>
+    <div class="more-space"></div>
+    <div class="block">
+      <div class="gray-area"></div>
+      <div class="description">
+        <div class="post">Multi-Post Stories Gain+Glory</div>
+        <div class="tags">
+          <div class="ruby">
+            <p class="font">Ruby on rails</p>
+          </div>
+          <div class="css">
+            <p class="font">css</p>
+          </div>
+          <div class="js">
+            <p class="font">JavScript</p>
+          </div>
+          <div class="html">
+            <p class="font">html</p>
+          </div>
         </div>
-        <div class="space"></div>
-      </section>
-    </main>
+        <div class="button-space">
+          <button type="button" class="project-text">See Project
+          </button>
+        </div>
+      </div>
+    </div>
+    <div class="block">
+      <div class="gray-area"></div>
+      <div class="description">
+        <div class="post">Multi-Post Stories Gain+Glory</div>
+        <div class="tags">
+          <div class="ruby">
+            <p class="font">Ruby on rails</p>
+          </div>
+          <div class="css">
+            <p class="font">css</p>
+          </div>
+          <div class="js">
+            <p class="font">JavScript</p>
+          </div>
+          <div class="html">
+            <p class="font">html</p>
+          </div>
+        </div>
+        <div class="button-space">
+          <button type="button" class="project-text">See Project
+          </button>
+        </div>
+      </div>
+    </div>
+    <div class="block">
+      <div class="gray-area"></div>
+      <div class="description">
+        <div class="post">Multi-Post Stories Gain+Glory</div>
```

Comment:
- Please, consider using an h2-h6 tag here, as this is the header and not a div item.